e2e/cypress/tests/integration/collapsed_reply_threads/global_threads_spec.js

```diff
@@ -45,9 +43,6 @@ describe('Collapsed Reply Threads', () => {
             // # Visit the channel
             cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
```

Comment:
This seemed to be unnecessary since page visit will be done as well in `beforeEach`. My only concern here is it will add up much in the overall test duration.